src/clj/ataru/applications/application_store.clj

```diff
@@ -117,6 +117,25 @@
     (unwrap-application application)
     (throw (ex-info "No existing form found when updating" {:secret secret}))))
 
+(defn- create-attachment-reviews
```

Comment:
Täs on jotain hassua, mikäli muistan speksin oikein. Eiks tuollanen attachment-review pitäis olla jokaista hakemuksen hakutoivetta kohden? Eli tyyliin
```
(map #(assoc review-base :hakukohde %)
     (or (seq (:hakukohde application))
         ["form"]))
```

Reply:
Totta :+1:

Reply:
Migraatioon pitää korjata sama homma.